.github/CHANGELOG.md

```diff
@@ -57,6 +69,10 @@
 
 This release contains contributions from (in alphabetical order):
 
+<<<<<<< HEAD
+
+=======
+>>>>>>> master
```

Comment:
Looks like some git stuff got merged in??

Reply:
@mariaschuld, could you undo this in the next template PR that is to be merged?

Reply:
I did, sorry for that! I also checked now if there are any <<<< or HEAD statements in path and it's clean.